src/extension_commands.ml

```diff
@@ -84,6 +84,27 @@ let switch_impl_intf =
   in
   command Extension_consts.Commands.switch_impl_intf handler
 
+module Dune_commands = struct
+  let open_current_dune_file =
+    let handler (_instance : Extension_instance.t) ~args:_ =
+      match Vscode.Window.activeTextEditor () with
+      | None ->
+        assert false
+        (* this command is available in the command palette only when a
+           ocaml/reason/mehnir/ocamllex files are open *)
+      | Some text_editor ->
+        let doc = TextEditor.document text_editor in
+        let uri = TextDocument.uri doc in
+        let dune_file_uri =
+          Uri.fsPath uri |> Path.of_string |> fun path ->
```

Comment:
Could you use let bindings here? It's weird to see `let x = foo` written as `foo |> fun x ->`.